bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoUnitSystem.java

```diff
@@ -63,17 +64,17 @@ public double convertAltitude(double value) {
 	
 	public BigDecimal convertRain(BigDecimal value) {
 		if (this == DEFAULT_UNIT_SYSTEM) {
-			return value;
+			return value.setScale(1, RoundingMode.HALF_UP);
```

Comment:
Replicated the same functionality as displayed in Netatmo apps, this is .1 of a millimeter